### PR TITLE
Refactor Switch

### PR DESCRIPTION
**Pull request by joshwooding**
Refactored Switch and updated its change handler.

```tsx
- const Controlled: ComponentStory<typeof Switch> = (args) => {
-   const [checked, setChecked] = useState(false);
-
-   const handleChange = (
-     _: ChangeEvent<HTMLInputElement>,
-     isChecked: boolean
-   ) => {
-     setChecked(isChecked);
-   };
-
-   return <Switch {...args} checked={checked} onChange={handleChange} />;
- };
+ const Controlled: ComponentStory<typeof Switch> = (args) => {
+   const [checked, setChecked] = useState(false);
+
+   const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+     setChecked(event.target.checked);
+   };
+
+   return <Switch {...args} checked={checked} onChange={handleChange} />;
+ };
```

Updated Switch styling

### CODE REVIEW

File: .idea/runConfigurations/_template__of_JavaScriptTestRunnerJest.xml [hunk 1]
```diff
@@ -1,9 +1,7 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="true" type="JavaScriptTestRunnerJest">
     <node-interpreter value="project" />
-    <node-options value="" />
```
Comment (libertymayc) [inline .idea/runConfigurations/_template__of_JavaScriptTestRunnerJest.xml:4]: are these changes supposed to be here

File: .changeset/hungry-plants-behave.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+"@salt-ds/lab": patch
+---
+
+Updated Switch's styling
+Refactored Switch and updated its change handler.
```
Comment (libertymayc) [inline .changeset/hungry-plants-behave.md:6]: should we say ControlLabel removed?

Comment (joshwooding) [reply .changeset/hungry-plants-behave.md:6]: Ah I forgot I did that, I was debating whether to keep it or delete it

**Review by libertymayc — APPROVED** [review]

**Review by Fercas123 — APPROVED** [review]

**Comment by bhoppers2008** [comment]
Figma component and guide published internally and externally.

**Pull request merged**